### DISCUSSION TITLE
MusicBrainz local database replica - indexed or nah?

### DISCUSSION THREAD

**Discussion by BourneLoser** [beetbox/beets#4121] [Q&A]
Hi everyone. I'd like to mirror the MusicBrainz db locally and have my Beets queries go to that local instance. I followed the link mentioned in https://beets.readthedocs.io/en/stable/reference/config.html#musicbrainz-options which led me to https://musicbrainz.org/doc/MusicBrainz_Server/Setup which states the disk requirement as "60GB+ of free disk space". The page doesn't mention database indexing which according to referenced link https://github.com/metabrainz/musicbrainz-docker#recommended-hardwarevm says "Disk Space: 150 GB (or 60 without indexed search)". Do Beets queries require the MusicBrainz database to be indexed? The additional 90GB requirement could mean the difference between using an internal vs external disk drive. Thanks

**Comment by sampsyo** [comment]
I have never used this myself, but I assume you want the database indices—otherwise, lookups and full-text searching could be very slow. It could be worth experimenting with, though.

**Comment by RollingStar** [comment]
The Musicbrainz wiki/IRC can probably help explain the differences.

https://musicbrainz.org/doc/Search

I am not familiar enough with how Beets does its searches.

> The direct search searches the database directly: it can only carry out simple keyword searches with no boolean logic, but it is always up-to-date.
